recipes/gia/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+#
+# TODO: Remove the following export when pinning is updated and we use
```

Comment:
Can this be removed now?

Reply:
Good point. I took this from johannes koester's rust-bio-tools a long time ago. I'll try and drop it next bump.

Reply:
Thanks for the recipe and merge!